.github/workflows/test-python.yml

```diff
@@ -62,6 +62,7 @@ jobs:
           dates
           datetimes
           db_functions
+          dbshell1
```

Comment:
When we'd like to use the same app name as Django (when tests in this repo are modeled off tests there), I haven't thought of a better way to differentiate than add something like "1". We could add something longer like "_mongo" but it's just more to type.